docs/source/asr/models.rst

```diff
@@ -128,6 +128,16 @@ You may find the example config files of Conformer-Transducer model with charact
 ``<NeMo_git_root>/examples/asr/conf/conformer/conformer_transducer_char.yaml`` and
 with sub-word encoding at ``<NeMo_git_root>/examples/asr/conf/conformer/conformer_transducer_bpe.yaml``.
 
+Fast-Conformer
+--------------
```

Comment:
Differentiate between fast conformer CTC and transducer.

Reply:
You may mention that it works with both the CTC and Transducer.

Reply:
We may add fastconformer to the list of models in the main nemo readme file.

Reply:
Not sure how to best do this. Separate head titles?  The fast conformer changes vs base are really not Transducer vs CTC specific.. the encoder is identical except the # of layers.